cypress/integration/dib-travel/my-travels/my-travels.spec.ts

```diff
@@ -0,0 +1,128 @@
+describe('My travels', () => {
+  beforeEach(() => {
+    cy.login();
+
+    cy.get('dib-navbar .home-page').contains(' My Travels ').click();
+  });
+
+  it('should display "My Travels" in the navbar', () => {
+    cy.get('dib-navbar .navbar__content').contains(' add trip ').should('exist');
+  });
+
+  it('should displays message that there are no active travels', () => {
+    cy.get('app-my-travels dib-travels-list .table-container.empty-list .empty-trips-section').should(
+      'contain',
+      " You don't have any active travels, but you can always create one "
+    );
+  });
+
+  it('should displays button to create new travel', () => {
+    cy.get('app-my-travels dib-travels-list .table-container.empty-list .dib-button-new').should(
+      'contain',
+      'Create new travel '
+    );
+  });
+
+  it('should check search for past travels', () => {
+    cy.get('app-my-travels .mat-tab-links').contains(' Past ').click();
+
+    cy.get('app-my-travels dib-travels-list dib-input input').type('CYQA');
```

Comment:
You can import first and last name from "User" model, you don't have to type hard coded 'CYQA'.